app/build.gradle

```diff
@@ -6,8 +6,8 @@ apply plugin: 'kotlin-kapt'
 android {
     compileSdkVersion 28
     defaultConfig {
-        applicationId "org.hackillinois.androidapp2019"
-        minSdkVersion 16
+        applicationId "org.hackillinois.android"
```

Comment:
Hopefully the app store will let us publish this under a new account? Let me know when you're ready to set up the beta version. I'd like to make a new android developer account with the email android@hackillinois.org